### DISCUSSION TITLE
How to enable font-feature-settings for a specific font-family

### DISCUSSION THREAD

**Discussion by nilsborg** [tailwindlabs/tailwindcss#9034] [Help] [answered]
Hey gang,
ran into a rather specific problem today: 
We're loading Cascadia Code (the variable one) like so:

```
@font-face {
    font-family: "Cascadia Code";
    src: url("/fonts/CascadiaCodeItalic.woff2") format("woff2");
    font-weight: 200 700;
    font-display: swap;
    font-style: italic;
    font-feature-settings: "ss01" 1; 
  }
```

and enable the font-family in `tailwind-config.js` here:

```
extend: {
  ...,
  fontFamily: {
    source: ["Cascadia Code", ...defaultTheme.fontFamily.mono],
  },
  ...,
}
```

and that works great expect that Chrome does not pick up the `font-feature-settings` in the font-face rule. Safari and Firefox do pick it up and that's great.

 So my question is if there's a Tailwind specific way to apply these `font-feature-settings` for only the `italic` and `font-source` combination of classes?

**Comment by wongjn** [comment]
For the simplest solution here since it is only 1 combination selector of `.italic.font-source`, I would write this out literally in your CSS:

```css
.italic.font-source {
  font-feature-settings: "ss01" 1; 
}
```
Otherwise, you could have a go at writing a custom plugin for it.

**Comment by adamwathan** [comment, marked as answer]
Opened a PR to allow you to configure default font-feature-settings for your configured custom fonts in the framework:

https://github.com/tailwindlabs/tailwindcss/pull/9039

Think this will help 👍

**Comment by nilsborg** [comment]
@wongjn Thank you for this super simple tip! I'll add that in until Adam's PR is merged!
@adamwathan thanks to you to for jumping on this so quickly!

**Comment by me-v2** [comment]
font-feature-settings
